media/CircleEditor/view.js

```diff
@@ -821,6 +825,24 @@ view.View = class {
                     if (this._page !== 'default') {
                         this.show('default');
                     }
+                    for (let idx = 0; idx < model.graphs.length; idx++) {
+                        model._graphs[idx]['_subgraphIdx'] = idx;
+                        for (let jdx = 0; jdx < model.graphs[idx].nodes.length; jdx++) {
+                            model._graphs[idx]._nodes[jdx]['_subgraphIdx'] = idx;
+                            if (builtinOperatorType
+                                    [model._graphs[idx]._nodes[jdx]._type.name.toUpperCase()] ===
```

Comment:
As far as I know, `CUSTOM` is defined in `builtinOperatorType` as 32.
Isn't it error when `builtinOperatorType` is undefined?

Reply:
And this is just optional, but it would be better to introduce a function like `builtinOperatorTypeAsNumber(string)` :)

Reply:
I was found a custom operator classification method in the netron model. I will use the if statement as like `model._graphs[idx]._nodes[jdx]._type?.category === 'custom'`. Thank you!!!